src/main/java/org/killbill/billing/plugin/adyen/core/KillbillAdyenNotificationHandler.java

```diff
@@ -204,6 +203,32 @@ private void handleNotification(final NotificationRequestItem item) {
         handleNotification(null, item);
     }
 
+    private void handleChargebackNotification(final NotificationRequestItem item, boolean success) {
+        final NotificationItem notification = new NotificationItem(item);
+        final DateTime utcNow = clock.getUTCNow();
+
+        final AdyenResponsesRecord originalRecord = getResponseRecord(item.getOriginalReference());
+        final UUID kbAccountId = UUID.fromString(originalRecord.getKbAccountId());
+        final UUID kbTenantId = UUID.fromString(originalRecord.getKbTenantId());
+        final UUID kbPaymentId = UUID.fromString(originalRecord.getKbPaymentId());
+
+        final CallContext context = new AdyenCallContext(utcNow, kbTenantId);
+        final Account account = getAccount(kbAccountId, context);
+
+        UUID kbPaymentTransactionId = null;
+        try {
+            final Payment payment = osgiKillbillAPI.getPaymentApi().createChargeback(account, kbPaymentId, notification.getAmount(), Currency.valueOf(notification.getCurrency()), null, context);
```

Comment:
> No Adyen doesn't give us one.

In that case, let's revert this change and let Kill Bill generate the UUID (in general, we prefer delegating the UUID generation to Kill Bill as much as possible, as it implements its own algorithm for performance reasons, see https://github.com/killbill/killbill/blob/master/api/src/main/java/org/killbill/billing/util/UUIDs.java).